openreview/venue/invitation.py

```diff
@@ -769,10 +769,14 @@ def set_recruitment_invitation(self, committee_name, options):
         else:
             print('current invitation', current_invitation.edit['note']['content'].get('reduced_load', {}))
             print('new invitation', reduced_load_dict)
-            if current_invitation.edit['note']['content'].get('reduced_load', {}) != reduced_load_dict:
-                print('update reduce load')
+            if ( current_invitation.edit['note']['content'].get('reduced_load', {}) != reduced_load_dict or
```

Comment:
can you split this in two different ifs?

if only one field has changed then I would like the edit to contain that single change instead of both. 

I would create the invitation object and set the value if each if sentence is true.